lib/provisioner/worker/plugins/automators/chef_solo_automator/chef_solo_automator.rb

```diff
@@ -315,8 +315,13 @@ def install_chef(inputmap)
           return
         rescue CommandExecutionError
           begin
-            ssh_exec!(ssh, "which apt-get && #{sudo} apt-get -q -y install chef", 'Attempting Chef install via apt-get')
-            return
+            candidate_version = ssh_exec!(ssh, "#{sudo} apt-cache policy chef | grep Candidate | awk '{ print $2}'", 'Retrieving candidate Chef version').first.chomp
+            log.debug "Found chef version #{candidate_version} available for install via package repositories"
+            # Bundled cookbooks require Chef 12 or later
+            if candidate_version.to_i >= 12
```

Comment:
Done... curious what requires 12.1 and not 12.0 though.  I was going off ``compat_resource``'s error message, which just said 12